src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -10532,6 +10532,13 @@ float Unit::GetUnitSpellCriticalChance(Unit* victim, SpellInfo const* spellProto
                 // Custom crit by class
                 switch (spellProto->SpellFamilyName)
                 {
+                    case SPELLFAMILY_PRIEST:
+                        // Renewed Hope should affect flash heal
+                        if (spellProto->SpellFamilyFlags[0] & 0x800 && spellProto->SpellIconID == 177)
```

Comment:
Are you sure this check is correct?
This selects 6788 - Weakened Soul but not Flash Heal